rts/lra/pom.xml

```diff
@@ -171,6 +171,7 @@
         <module>lra-filters</module>
         <module>lra-annotation-checker</module>
         <module>lra-test</module>
+        <module>lra-service-base</module>
```

Comment:
I think the `lra-service-base` module should be moved to the top of the list. I think maven runs compilation/installation in sequence by order defined in pom.xml. If the `lra-service-base` is used by other modules it should be compiled before other ones.

Reply:
that's not totally true. Maven first checks dependencies of the modules so if there is dependency in lra-client to lra-service-base for instance, lra-service-base must be build before lra-client so it is buildable in clean environment. But I can definitely make this change, however I find it unnecessary and CI is already running for this and force push would retrigger it:) @ochaloup you want to update this?

Reply:
ok, then it's my ignorance. sorry for that. I have no issue with this in such case.